docs/user-guide/commonalities.md

```diff
@@ -0,0 +1,883 @@
+in OpenShift](https://kubernetes.io/docs/concepts/storage/volumes/):
+`ConfigMaps`, `Secrets`, `NFS` shares, etc.
+
+These `extraMounts` can be defined at different levels: Deployment, Service, and
+Component. For the higher levels we can define where these should be applied by
+using the propagation functionality, otherwise they will propagate to all
+available resources below; so at the deployment level they would go to all the
+services (cinder, glance, manila, neutron, horizon…), at the service level they
+would go to all the individual components (e.g.: cinder api, cinder scheduler,
+cinder volume, cinder backup), and at the component level it can only go to the
+component itself.
+
+This the the general structure of the `extraMounts` which contains a list of
+“mounts”:
+
+```
+extraMounts:
+  - name: <extramount-name> [1]
+    region: <openstack-region> [2]
+    extraVol:
+      - propagation: [3]
+        - <location>
+        extraVolType: <Ceph | Undefined> [4]
+        volumes: [5]
+        - <pod-volume-structure>
+        mounts: [6]
+        - <pod-mount-structure>
+```
+
+- \[1\]: Optional name of the extra mount, not of much significance since it’s not
+referenced from other parts of the manifest.
+
+- \[2\]: Optional name of the OpenStack region where this `extraMount` should be
+applied.
+
+- \[3\]: Location to propagate this specific `extraMount`.
+
+  There are multiple names that can be used each with different granularity:
+
+  * Service names: `Glance`, `Cinder`, `Manila`
+  * Component names: `CinderAPI`, `CinderScheduler`, `CinderVolume`,
+    `CinderBackup`, `GlanceAPI`, `ManilaAPI`, `ManilaScheduler`,  `ManilaShare`
+  * Back-end names: For cinder it would be any name in the `cinderVolumes` map.
+    In the examples we’ve shown before it could have been `fibre_channel` or
+    `iscsi` or `nfs`.
+  * Step name: `DBSync`
+
+  **⚠ Attention:** Defining a broad propagation at a high level can result in
+  the mount happening in many different components. For example using `DBSync`
+  propagation at the deployment level will propagate to the cinder, glance, and
+  manila DB sync pods.
+
+- \[4\] Optional field declaring the type of propagation. It’s possible values
+are `Ceph` and `Undefined`, so in reality it only makes sense to set it when
+used to propagate Ceph configuration files.
+
+- \[5\]: List of volume sources, where volumes refers to the broader term of a
+“volume” as understood by OpenShift.
+
+  This has the exact same structure as the `volumes` section in a `Pod` and is
+  dependent on the type of volume that is being defined, as it is not the same
+  to reference an NFS location as it is to reference a secret.
+
+  The name used here is very important, as it will be used as a reference in the
+  `mounts` section.
+
+  An example of the excerpt of a `volumes` element that defines that the volume
+  is a secret called `ceph-conf-files`:
+
+  ```
+       - name: ceph
+         projected:
```

Comment:
I was using projected because then it's easier for deployments with multiple Ceph clusters.

I'll change it and leave the multiple Ceph clusters documentation for a follow up.